### PR TITLE
contract update for new feature

### PR DESCRIPTION
**Pull request by tomklecha**

### CODE REVIEW

**Comment by tomklecha** [comment]
like I created this PR like that, because I have concept how to make it work - i done small test run. Problem was how to test interactor. Or should I also show notification on successful added question to the list ? that would be testable then

File: features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt [hunk 1]
```diff
@@ -30,10 +32,18 @@ internal interface QuestionsContract {
         fun onBackPressed()
 
         fun fetchNextCategoryQuestions()
+
+        fun calculateScore(question: QuestionsModelDetails, isChecked: Boolean)
```
[the remark this answers is not in the document]
Comment (tomklecha) [reply features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt:36]: maybe it is more naming convention - > calculate score will always add/remove the checked/unchecked element to interactor list

File: features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt [hunk 2]
```diff
@@ -30,10 +32,18 @@ internal interface QuestionsContract {
...
+
+        fun getPurityScore()
```
[the remark this answers is not in the document]
Comment (tomklecha) [reply features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt:38]: getPurityScore will trigger interactor to summarize all score and return in to view.showScore method

File: features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Canato) [reply features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt:36]: Why we want to calculate for each check/uncheck? 
How could we improve this logic?

File: features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Canato) [reply features/questions/src/main/java/com/can_apps/questions/core/QuestionsContract.kt:38]: you are not "getting" anything, has not return.

When you plan to call `getPurityScore` ?

**Comment by Canato** [comment]
We probably should have a call before you keep going @tomklecha

**Comment by tomklecha** [comment]
could we do the call 8th of december?
we already had plans for next days and after that I will be already in Poland, so I will be free to go

**Pull request closed without merging**